lib/Layout/Body.js

```diff
@@ -4,7 +4,7 @@ import { node, string } from 'prop-types';
 export function Body({ children, className }) {
   return (
     <div
-      className={`ql-scroll-container relative py-10 w-full flex flex-col flex-1 min-h-0 overflow-y-auto container ${className}`}
+      className={`ql-scroll-container relative w-full flex flex-col flex-1 min-h-0 overflow-y-auto container ${className}`}
```

Comment:
I removed this as a small change in this PR because the body should not choose padding!